apps/meteor/client/views/admin/apps/AppDetailsPage/tabs/AppStatus/TooltipOnHover.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, PositionAnimated, AnimatedVisibility } from '@rocket.chat/fuselage';
+import React, { RefObject, useRef, useState, ReactElement } from 'react';
+
+type TooltipOnHoverProps = {
+	element: ReactElement;
+	tooltip: ReactElement;
+};
+
+export const TooltipOnHover = ({ element, tooltip }: TooltipOnHoverProps): ReactElement => {
+	const ref = useRef<Element>();
```

Comment:
type it as:
`const ref: RefObject<Element> = useRef(null);`
to avoid the assertion on line 20.